### PR TITLE
added bio for Bertrand Thirion

### PR DESCRIPTION
**Pull request by bthirion**

### CODE REVIEW

File: content/material.md [hunk 1]
```diff
@@ -420,6 +420,7 @@ Given the time restrictions, we will focus on `biological agents`, ie `human par
...
 
 **Bertrand Thirion**
+Bertrand Thirion is senior researcher in the MIND team, part of Inria research institute, Saclay, France, that develops statistics and machine learning techniques for brain imaging. He contributes both algorithms and software, with a special focus on functional neuroimaging applications. He is involved in the Neurospin, CEA neuroimaging center, one of the leading high-field MRI for brain imaging places. From 2018 to 2021, Bertrand Thirion has been the head of the DATAIA Institute that federates research on AI, data science and their societal impact in Paris-Saclay University. In 2020, he has recently been appointed as member of the expert committee in charge of advising the government during the Covid-19 pandemic. In 2021, he has become the Head of science (délégué scientifique) of the Inria Saclay-Île-de-France research center. Bertrand Thirion is PI of the [Karaib AI Chair](http://project.inria.fr/karaib) of the [Individual Brain CHarting](http://project.inria.fr/ibc) project.
```
Comment (htwangtw) [inline content/material.md]:
```suggestion
Bertrand Thirion is senior researcher in the MIND team, part of Inria research institute, Saclay, France, that develops statistics and machine learning techniques for brain imaging. He contributes both algorithms and software, with a special focus on functional neuroimaging applications. He is involved in the Neurospin, CEA neuroimaging center, one of the leading high-field MRI for brain imaging places. From 2018 to 2021, Bertrand Thirion has been the head of the DATAIA Institute that federates research on AI, data science and their societal impact in Paris-Saclay University. In 2020, he has recently been appointed as member of the expert committee in charge of advising the government during the Covid-19 pandemic. In 2021, he has become the Head of science (délégué scientifique) of the Inria Saclay-Île-de-France research center. Bertrand Thirion is PI of the [Karaib AI Chair](http://project.inria.fr/karaib) of the [Individual Brain Charting](http://project.inria.fr/ibc) project.
```

**Review by htwangtw — COMMENTED** [review]
Just one typo!

**Comment by bthirion** [comment]
Thx !

**Comment by lunebellec** [comment]
LGTM!

**Pull request merged**